docs/specviz/import_data.rst

```diff
@@ -33,10 +35,19 @@ Here, users can select the loaded data set to be visualized in the viewer.
 
 .. image:: img/data_selected_1.png
 
-Importing data via the API
---------------------------
-Alternatively, if users are working in a coding environment like a Jupyter notebook, they have access to the Specviz helper class API. Using this API, users can load data into the application through code.
-Below is an example of importing the Specviz helper class, creating a :class:`~specutils.Spectrum1D` using randomly generated data, and loading that :class:`~specutils.Spectrum1D` object into the Specviz application::
+.. _api-import:
+
+Loading data via the API
+------------------------
+Alternatively, if users are working in a coding environment like a Jupyter notebook, they have access to the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class API. Using this API, users can load data into the application through code.
+Below is an example of importing the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class, creating a :class:`~specutils.Spectrum1D` object from a data file via the :func:`~specutils.Spectrum1D.read` method::
+
+    >>> from specutils import Spectrum1D
+    >>> spec1d = Spectrum1D.read("/path/to/data/file") #doctest: +SKIP
+    >>> specviz = SpecViz() #doctest: +SKIP
+    >>> specviz.load_spectrum(spec1d)
```

Comment:
Seems like this line is failing in the tests. Does Specviz need to be imported here?

Reply:
Ah, thanks - no, it looks like I just forgot to add the `` #doctest: +SKIP`` on that line!